src/Tools/Directory.Build.props

```diff
@@ -6,7 +6,7 @@
     <OutputType>Exe</OutputType>
     <IsPackable>true</IsPackable>
     <PackAsTool>true</PackAsTool>
-    <PackAsToolShimRuntimeIdentifiers>win-x64;win-x86;osx-x64</PackAsToolShimRuntimeIdentifiers>
+    <PackAsToolShimRuntimeIdentifiers>win-x64;win-x86;win-arm;win-arm64;osx-x64</PackAsToolShimRuntimeIdentifiers>
```

Comment:
Why ARM64 windows? Is it even a supported scenario?

Reply:
Just to be complete.  We will have to support Windows ARM64 soon. I have an issue tracking it.

Reply:
Removed